internal/messagequeue/messagequeue.go

```diff
@@ -34,6 +32,11 @@ const (
 	maxPriority = math.MaxInt32
 	// sendMessageDebounce is the debounce duration when calling sendMessage()
 	sendMessageDebounce = time.Millisecond
+	// when we reach sendMessaageCuttoff wants/cancels, we'll send the message immediately.
+	sendMessageCuttoff = 100
```

Comment:
Too big?

Reply:
I would actually make `sendMessageCutoff` bigger and `sendMessageMaxDelay` smaller